libqtile/layout/max.py

```diff
@@ -44,17 +45,21 @@ def add(self, client):
         return super().add(client, 1)
 
     def configure(self, client, screen_rect):
-        if self.clients and client is self.clients.current_client:
+        if not self.only_focused or (self.clients and client is self.clients.current_client):
+            idx = self.clients.current_index
+            order = (self.clients[idx+1:] + self.clients[:idx+1])
             client.place(
                 screen_rect.x,
                 screen_rect.y,
                 screen_rect.width,
                 screen_rect.height,
                 0,
-                None
+                None,
+                z=order.index(client)
```

Comment:
Won't `order.index(client)` always equal `len(self.clients)`, which is easier to use than line 50?

Reply:
`len(self.clients) - 1`, in fact

Reply:
Oh, no, I totally misread the code. Never mind - this makes sense.